database_pregel.go

```diff
@@ -0,0 +1,182 @@
+	PregelJobStateStoring PregelJobState = "storing"
+	// PregelJobStateDone  - The execution is done. In version 3.7.1 and later, this means that storing is also done.
+	// In earlier versions, the results may not be written back into the collections yet. This event is announced in the server log (requires at least info log level for the pregel log topic).
+	PregelJobStateDone PregelJobState = "done"
+	// PregelJobStateCanceled  - The execution was permanently canceled, either by the user or by an error.
+	PregelJobStateCanceled PregelJobState = "canceled"
+	// PregelJobStateFatalError - The execution has failed and cannot recover.
+	PregelJobStateFatalError PregelJobState = "fatal error"
+	// PregelJobStateInError - The execution is in an error state. This can be caused by DB-Servers being not reachable or being non-responsive.
+	// The execution might recover later, or switch to "canceled" if it was not able to recover successfully.
+	PregelJobStateInError PregelJobState = "in error"
+	// PregelJobStateRecovering - (currently unused): The execution is actively recovering and switches back to running if the recovery is successful.
+	PregelJobStateRecovering PregelJobState = "recovering"
+)
+
+type PregelJob struct {
+	// The ID of the Pregel job, as a string.
+	Id string `json:"id"`
+	// The algorithm used by the job.
+	Algorithm PregelAlgorithm `json:"algorithm,omitempty"`
+	// The date and time when the job was created.
+	Created time.Time `json:"created,omitempty"`
+	// The date and time when the job results expire.
+	// The expiration date is only meaningful for jobs that were completed, canceled or resulted in an error.
+	// Such jobs are cleaned up by the garbage collection when they reach their expiration date/time.
+	Started time.Time `json:"started,omitempty"`
+	// The TTL (time to live) value for the job results, specified in seconds. The TTL is used to calculate the expiration date for the job’s results.
+	TTL uint64 `json:"ttl,omitempty"`
+	// The state of the execution.
+	State PregelJobState `json:"state,omitempty"`
+	// The number of global supersteps executed.
+	Gss uint64 `json:"gss,omitempty"`
+	// The total runtime of the execution up to now (if the execution is still ongoing).
+	TotalRuntime float64 `json:"totalRuntime,omitempty"`
+	// The startup runtime of the execution. The startup time includes the data loading time and can be substantial.
+	StartupTime float64 `json:"startupTime,omitempty"`
+	// The algorithm execution time. Is shown when the computation started.
+	ComputationTime float64 `json:"computationTime,omitempty"`
+	// The time for storing the results if the job includes results storage. Is shown when the storing started.
+	StorageTime float64 `json:"storageTime,omitempty"`
+	// Computation time of each global super step. Is shown when the computation started.
+	GssTimes []float64 `json:"gssTimes,omitempty"`
```

Comment:
```suggestion
	GSSTimes []float64 `json:"gssTimes,omitempty"`
```
go guidelines naming ^